tox.ini

```diff
@@ -13,13 +13,16 @@ envlist =
     deprecated-pet-tests
 
 [testenv]
+package = editable
 passenv = *
 lint_folders =
     "{toxinidir}/examples/" \
     "{toxinidir}/src/" \
     "{toxinidir}/tests/" \
     "{toxinidir}/docs/src/"
 
+pytest_args=--durations=0 --durations-min=1.0
+
 [testenv:lint]
 description = Run linters and type checks
```

Comment:
Are these changes needed?

Reply:
Partially =) I think that what we don't see very explicitly will be ignored, so I like always printing which tests are being too slow. But let me split this into a separate PR so it can be discussed on it's own

Reply:
Partially =) I think that what we don't see very explicitly will be ignored, so I like always printing which tests are being too slow. But let me split this into a separate PR so it can be discussed on it's own